modules/central-configuration-options-operator.adoc

```diff
@@ -114,6 +114,9 @@ If no PVC with the given name exists, it is created. The default value is `centr
 |`central.db.persistence.persistentVolumeClaim.storageClassName`
 | The name of the storage class to use for the PVC. If your cluster is not configured with a default storage class, you must provide a value for this parameter.
 
+|`central.db.connectionPoolSize`
+| Use this parameter to override the default connection pool size, which sets both `minConnections` and `maxConnections` between Central and Central DB.
```

Comment:
I took a look at the Jira ticket, but am not sure what specific values the user might want to use here and in what circumstance. Do we need to document that? Maybe this is something that is typically known by users and we don't, I am just checking.

Reply:
Also, do we want to document the default value? (I guess it's 200 connections)

Reply:
The default values are 10 (for `minConnection`) and 90 (for `maxConnections`). I'm not sure if we should mention this in the docs, because:
* if we change this in the code we might forget to update the docs
* the default values can be seen in the OpenShift UI when creating a new Central

These are the number of connections that Central can open. But Central DB itself has a default limit of max 200 connections, and a value of `maxConnections` greater than that will be ignored. This max value can be changed by providing a custom `postgresql.conf` file via the `central.db.configOverride` parameter - which I just discovered that it's not documented. Also, this applies only to the operator-managed Central DB; in case of external DBs it's part of the external DB configuration (and we don't have control over that).

It's hard to say what values the users should use here. In case of scaling issues for very large clusters, users might benefit from increasing maxConnections. But it's pretty much an advanced use case for which we can't offer generic guidelines. So maybe we should write something along these lines instead: don't change these unless you know what you're doing. :)